include/ebpf_structs.h

```diff
@@ -69,6 +69,11 @@ typedef struct _ebpf_map_definition_in_file
      * file is the inner map template.
      */
     uint32_t inner_map_idx;
+
+    // id is the identifier for a map template.
```

Comment:
I don't think this comment style will show up in doxygen docs like lines 61-65 do

Reply:
fixed.